scene/resources/environment.cpp

```diff
@@ -346,13 +346,40 @@ bool Environment::is_ssao_enabled() const {
 	return ssao_enabled;
 }
 
-void Environment::set_ssao_radius(float p_radius) {
-	ssao_radius = p_radius;
+void Environment::set_ssao_noise_tolerance(float p_noise_tolerance) {
+	ssao_noise_tolerance = p_noise_tolerance;
 	_update_ssao();
 }
 
-float Environment::get_ssao_radius() const {
-	return ssao_radius;
+float Environment::get_ssao_noise_tolerance() const {
+	return ssao_noise_tolerance;
+}
+
+void Environment::set_ssao_blur_tolerance(float p_blur_tolerance) {
+	ssao_blur_tolerance = p_blur_tolerance;
+	_update_ssao();
+}
+
+float Environment::get_ssao_blur_tolerance() const {
+	return ssao_blur_tolerance;
+}
+
+void Environment::set_ssao_upsample_tolerance(float p_upsample_tolerance) {
+	ssao_upsample_tolerance = p_upsample_tolerance;
+	_update_ssao();
+}
+
+float Environment::get_ssao_upsample_tolerance() const {
+	return ssao_upsample_tolerance;
+}
+
+void Environment::set_ssao_rejection_radius(float p_rejection_radius) {
+	ssao_rejection_radius = p_rejection_radius;
+	_update_ssao();
+}
+
+float Environment::get_ssao_rejection_radius() const {
```

Comment:
Is it very common to modify these values? If the defaults work fine for say 99% of cases, I suggest moving them to project settings.